scipy/signal/setup.py

```diff
@@ -1,5 +1,28 @@
 from scipy._build_utils import numpy_nodepr_api
 import os
+import sys
+
+def process_tempita(fromfile):
```

Comment:
I think we should put this in `scipy._build_tools` or `tools/`, and vendor `tempita` rather than do the try-except dance.

Reply:
Moved it all to _build_utils for now. Maybe push off vendoring until we really need it --- I rather strongly suspect this try-except dance is all dead code as long as cython bundles tempita.